### PR TITLE
Logging Server updated

### PR DESCRIPTION
**Pull request by Daffurn**

### CODE REVIEW

File: database.py [hunk 1]
```diff
@@ -17,9 +17,12 @@ class Log(Base):
     """ Table for storing the log information. """
     __tablename__ = 'log'
     id = Column(String(250), primary_key=True)
+    userID = Column(String(250), nullable=False)
+    trigger = Column(String(250), nullable=False)
     event = Column(String(250), nullable=False)
     value = Column(String(250), nullable=False)
     time = Column(String(250), nullable=False)
+    type = Column(String(250), nullable=False)
```
Comment (DanChirtoaca) [inline database.py]: some documentation regarding all the columns is necessary

File: main.py [hunk 2]
```diff
@@ -1,7 +1,12 @@
 from flask import Flask, render_template, request
 from database import session_scope, Log
+from flask_cors import CORS, cross_origin
 app = Flask(__name__)
+CORS(app)
 
+with session_scope() as db_session:
+    x = db_session.query(Log).count() + 1
+
```
Comment (DanChirtoaca) [inline main.py]: x is a counter I assume? Do specify what this piece of code does for you and why is it needed here

File: main.py [hunk 3]
```diff
@@ -11,17 +16,19 @@ def index():
...
 
 @app.route('/submit', methods=['GET', 'POST'])
 def submit():
+    global x
     data = request.get_json()
-    entry = Log(id=data['id'], event=data['event'], value=data['value'], time=data['time'])
-    with session_scope() as db_session:
-        db_session.add(entry)
-    return "ok"
+    for i in data:
```
Comment (DanChirtoaca) [inline main.py]: data comes in for multiple users at once? Document this please!

File: post.py [hunk 4]
```diff
@@ -19,7 +19,7 @@
         "trigger": "switch",
         "value": "23",
         "time": "2017-05-29T15:20:28.661Z",
-        "type": "fucku"}
```
Comment (DanChirtoaca) [inline post.py]: typo - sure :D

**Comment by mircealungu** [comment]
one more thing -- to deploy this you need to define a package that can be installed on the target system. you need to have all the code inside a package and a setup.py that installs that package. 

for an example, please look at: https://github.com/mircealungu/automatic-monitoring-dasboard/tree/development (where the package is called dashboard)

**Comment by Daffurn** [comment]
I've implemented the requested changes. Have a look and let me know what you think

**Pull request closed without merging**